gradio/chat_interface.py

```diff
@@ -338,8 +339,21 @@ def _setup_example_messages(
                     example_message["files"] = example.get("files", [])
                 if example_labels:
                     example_message["display_text"] = example_labels[index]
-                if example_icons:
-                    example_message["icon"] = example_icons[index]
+                if multimodal:
```

Comment:
```suggestion
                if self.multimodal:
```

Reply:
I would instead do this. 

You'll also need to remove the `@staticmethod` decorator from the `_setup_example_messages` method.